### ISSUE TITLE
[Adaptive Cards iOS - FoodOrder-]: 'How would you like your steak prepared?' label is not associated with its Radio button options. (Screen reader)

### ISSUE THREAD

**Issue by PankajBhojwani** [microsoft/AdaptiveCards#7108]
### Target Platforms

iOS

### SDK Version

iOS 14.6

### Application Name

Adaptive Cards iOS

### Problem Description

[33779637](https://microsoft.visualstudio.com/OS/_workitems/edit/33779637)

### Card JSON

```JSON
FoodOrder.json
```

**Comment by vagpt** [comment]
This issue is not repro on the below environment, hence closing this issue.

Build Version -1.0(2.4.0-beta.20220718.1)
